component-classes/index.js

```diff
@@ -207,14 +207,14 @@ export const expandable = {
   expandableBleed: box.bleed,
   chevron: 'inline-block align-middle i-text-$color-expandable-icon',
   chevronNonBox: 'relative left-8',
-  chevronBox: 'absolute right-16',
+  chevronBox: '',
   chevronTransform: 'transform transition-transform transform-gpu ease-in-out',
   chevronExpand: '-rotate-180',
   chevronCollapse: 'rotate-180',
   expansion: 'overflow-hidden',
   expansionNotExpanded: 'h-0 invisible',
   button: buttonReset + ' hover:underline focus-visible:underline',
-  buttonBox: 'w-full text-left relative inline-flex items-center ' + box.box,
+  buttonBox: 'w-full text-left relative inline-flex items-center space-between ' + box.box,
```

Comment:
I think it should be `justify-between` (see the unocss output snapshot [here](https://github.com/warp-ds/drive/blob/next/test/__snapshots__/position.js.snap#L482))

Reply:
You're totally right! My brains got scrambled copying the fix over from my test env! 🤦